dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -3,9 +3,13 @@
 class Api::V1::UsersController < Api::V1::JSONApiController
   before_action :load_user
   skip_before_action :verify_authenticity_token
-  skip_before_action :load_user, only: [:current, :netsim_signed_in, :post_sort_by_family_name, :post_show_progress_table_v2, :get_current_permissions, :post_disable_lti_roster_sync]
+  skip_before_action :load_user, only: [:current, :netsim_signed_in, :post_sort_by_family_name, :post_show_progress_table_v2, :get_current_permissions, :post_disable_lti_roster_sync, :update_ai_tutor_access]
   skip_before_action :clear_sign_up_session_vars, only: [:current]
 
+  private def to_bool(val)
```

Comment:
It seems weird that a utility function like this doesn't already exist. I couldn't figure out where a generic util like this should live in the dashboard code... so far, we've defined it on a per-controller level (ex., the Amazon future engineer controller has this same thing), which is not ideal. The to_bool method used in this file in other places fails if you pass it a boolean and not a string, which seems... not ideal. I'd prefer this handles both!

Reply:
Too bad to_bool doesn't already do this.... but you can't go wrong using `ActiveModel::Type::Boolean.new.cast` which is built-in to Rails 👍